openmeetings-service/src/main/java/org/apache/openmeetings/service/mail/template/InvitationTemplate.html

```diff
@@ -16,20 +16,172 @@
+                  <!-- START CONTAINER -->
+                  <table role="presentation" class="main" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background: #ffffff; border-radius: 3px; width: 100%;" width="100%">
+                      <!-- START MAIN MSG -->
+                      <tr>
+                          <td class="wrapper" style="font-family: 'Montserrat', sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;" valign="top">
+                              <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;" width="100%">
+                                  <tr>
+                                      <td style="font-family: 'Montserrat', sans-serif; font-size: 14px; vertical-align: top;" valign="top">
+                                        <!-- INVITATION LABEL -->
+                                          <span wicket:id="titleLbl" style="font-family: 'Montserrat', sans-serif; font-size: 20px; font-weight: normal; margin: 0; margin-bottom: 15px;"></span>
+                                          <!-- END INVITATION LABEL -->
+                                          <br /><br />
+                                          <!-- INVITATION MSG -->
+                                          <!-- <span wicket:id="titleLbl" style="font-family: 'Montserrat', sans-serif; font-size: 14px; font-weight: bold; margin: 0; margin-bottom: 20px;"></span><br /><br /> -->
+                                          <span wicket:id="userLbl"></span> <span wicket:id="user"></span><br/>
+                                          <span wicket:id="messageLbl"></span> <span wicket:id="message"></span><br/>
+                                          <!-- END INVITATION MSG -->
+                                          <br />
+                                          <table
+                                              role="presentation"
+                                              border="0"
+                                              cellpadding="0"
+                                              cellspacing="0"
+                                              class="btn btn-primary"
+                                              style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; box-sizing: border-box; width: 100%;"
+                                              width="100%"
+                                          >
+                                              <tbody>
+                                                  <tr>
+                                                      <td align="left" style="font-family: 'Montserrat', sans-serif; font-size: 14px; vertical-align: top; padding-bottom: 15px;" valign="top">
```

Comment:
would it work for RTL languages?

Reply:
OK, I'll confirm that

Reply:
Yeah, it works for RTL languages.

Reply:
can I have screen shots for RTL please

Reply:
Yeah, Mr. @ali-alhaidary
![4RTL](https://github.com/apache/openmeetings/assets/52314615/4b2b2861-0a18-4781-8fbd-6b6da8321fb2)

Reply:
very good, thanks...

Reply:
You're welcome